doc/source/conf.py

```diff
@@ -77,10 +77,12 @@
 }
 
 intersphinx_mapping = {
-    'ansible': ('https://docs.ansible.com/ansible/devel/', None),
+    'ansible': ('https://docs.ansible.com/ansible/latest/', None),
```

Comment:
FTR: Unlike commonly known practice, `latest` in Ansible docs corresponds to the stable version, not to the latest state of the main Git branch. Is this your intention? Just checking...

Reply:
Yep, stable seems to be more suitable place to link to.